src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  let DAYS_RATE = days * PRICE_PER_DAY;
```

Comment:
```suggestion
  const basicPrice = days * PRICE_PER_DAY;
```
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/96409563/b6b7e7d8-45e5-4a2a-bb60-e8017baa93f3)
Also, it's a calculated variable, so should be written in camelCase

Reply:
Thank you! I'll take this into account:)